src/openms/source/FORMAT/SqliteConnector.cpp

```diff
@@ -60,15 +60,19 @@ namespace OpenMS
   {
     bool found = false;
 
-    sqlite3_stmt * xcntstmt;
-    SqliteConnector::executePreparedStatement(db, &xcntstmt, "PRAGMA table_info(" + tablename + ")");
+    sqlite3_stmt* xcntstmt;
+    prepareStatement(db, &xcntstmt, "PRAGMA table_info(" + tablename + ")");
 
     // Go through all columns and check whether the required column exists
     sqlite3_step(xcntstmt);
-    while (sqlite3_column_type( xcntstmt, 0 ) != SQLITE_NULL)
+    while (sqlite3_column_type(xcntstmt, 0) != SQLITE_NULL)
     {
-      String name = String(reinterpret_cast<const char*>(sqlite3_column_text( xcntstmt, 1 )));
-      if (colname == name) {found = true;}
+      String name = String(reinterpret_cast<const char*>(sqlite3_column_text(xcntstmt, 1)));
```

Comment:
reinterpret_cast required here?

Reply:
Shouldn't be - well spotted! I'm removing that and another one in the file.